kpagacz/sudoku-app
# 15 sudoku list

How to test:
Go to sudoku list, see how it looks.
Click on some sudoku and look at id printed on console.
To test pagination, you can change "itemsPerPage" variable in SudokuListController to some smaller number to have more pages.

closes #15
closes #18
